portal/server/routes/contract/bond/index.js

```diff
@@ -268,7 +304,34 @@ router.post('/contract/:_id/bond/:bondId/save-go-back', provide([BOND]), async (
   const { _id: dealId, bondId, userToken } = requestParams(req);
   const { bond } = req.apiData.bond;
 
-  const modifiedBody = feeFrequencyField(req.body, bond);
+  const allowedProperties = [
```

Comment:
This list is long due to any of the three forms in the process being able to call this endpoint and function. Seeing as you would step through the forms anyway and have access to all of them if you have access to one means this isn't very dangerous to allow people to cross enter details from other forms. However due to different hidden inputs that depend on what stage the facility is in we might want to lock this down further and see what form is being submitted and only allow fields from that form.

I'm not sure of the risk of being able to update say a the requestedCoverStartDate-month when the bond isn't issued and so only ukefGuaranteeInMonths should allowed to be submitted - thoughts?